frontend/packages/dev-console/integration-tests/views/dev-perspective.view.ts

```diff
@@ -18,7 +18,7 @@ export enum Perspective {
 }
 
 export const pageSidebar = $('#page-sidebar .pf-c-nav .pf-c-nav__list');
-export const sideHeader = $('#page-sidebar .oc-nav-header h1');
+export const sideHeader = $('#page-sidebar .oc-nav-header h2');
```

Comment:
Existing code, but it would be better to give these `data-test` attributes.

Reply:
Updated the code to use `data-test-id` now.